app/jobs/activity_tracking_job.rb

```diff
@@ -20,6 +20,27 @@ def perform(action:, trackable_id:, trackable_type:, user_id:)
       user: user.email
     )
 
-    trackable.notify(action) if trackable.respond_to?(:notify)
+    if trackable.respond_to?(:notify)
+      trackable.notify(action)
+      broadcast_notifications(trackable)
+    end
+  end
+
+  private
+
+  def broadcast_notifications(trackable)
+    project = trackable.commentable.project
```

Comment:
not all trackables have a commentable ?
If in the future we want to create notifications when an issue is created, trackable would be an issue but an issue does not have a commentable